js/pom.xml

```diff
@@ -27,8 +27,11 @@
     <properties>
         <node.version>v22.11.0</node.version>
         <pnpm.version>9.14.4</pnpm.version>
-        <!-- The JavaScript projects use the non-standard 'src' folder for their sources, therefore, name it here explicitly -->
+        <!-- The JavaScript projects use non-standard folders for their sources, therefore, name it here explicitly -->
         <maven.build.cache.input.1>src</maven.build.cache.input.1>
+        <maven.build.cache.input.2>public</maven.build.cache.input.2>
+        <maven.build.cache.input.3>scripts</maven.build.cache.input.3>
+        <maven.build.cache.input.4>maven-resources-community</maven.build.cache.input.4>
```

Comment:
Should we not also be adding `maven-resources` here? Or is this handled by specifying it as a direct resource in the POM? If so can't we change `maven-resources-community` to be a direct resource too (rather than a config)?

Reply:
When debugging it, I found that it picked up the `maven-resources` folder automatically from the `resources` configuration, so no need to override that.

Reply:
Yeah, I am wondering if we can also get `maven-resources-community` to be picked up automatically by providing it as a direct resource in the profile rather than a configuration for the plugin.

Reply:
Pushed one more commit that simplifies that.